crates/noirc_evaluator/src/errors.rs

```diff
@@ -21,6 +21,16 @@ impl RuntimeError {
     pub fn new(kind: RuntimeErrorKind, location: Option<Location>) -> RuntimeError {
         RuntimeError { location, kind }
     }
+
+    pub fn merge_location(
```

Comment:
can you add a comment to this, and would it make sense to call location1 `lhs_location` and location2 `rhs_location` ?

Thinking of ways to give this function more context

Reply:
Done.
lhs and rhs do not make sense to me, they come from instructions, not from some operands.
I renamed to a  and b which allows to have the function signature on one line and I think it is more readable, but I am open to better ideas.